packages/ra-core/src/store/localStorageStore.ts

```diff
@@ -93,11 +93,12 @@ export const localStorageStore = (version: string = '1'): Store => {
             }
         },
         getItem<T = any>(key: string, defaultValue?: T): T {
-            const valueFromStorage = tryParse(
-                getStorage().getItem(`${RA_STORE}.${key}`)
-            );
+            const valueFromStorage = getStorage().getItem(`${RA_STORE}.${key}`);
+
             // eslint-disable-next-line eqeqeq
```

Comment:
There is a configuration option for that rule that will never warn about `== null`

Reply:
nice! let's enable it in another Pr (I didn't add this line)